control/veda_auth.yaml

```diff
@@ -16,6 +16,14 @@ edx_s3_ingest_bucket:
 edx_s3_endpoint_bucket:
 edx_cloudfront_prefix:
 
+# transcript bucket config
+transcript_bucket_access_key:
+transcript_bucket_secret_key:
```

Comment:
@muhammad-ammar use `instance_config.yaml` in the root directory instead. We're trying to get to a unified config file with `edx-video-worker`